### ISSUE TITLE
question about 'tokenizer.py'

### ISSUE THREAD

**Issue by tanyz0208** [anananan116/MBGen#3]
Greetings, fantastic work! I have some questions, and hopefully can get some help from you.
As for the function 'generate_embedding(self, config)' in tokenizer.py, it tries to load 'embedding.pkl' located in './tokenizer/'. However, it seems there's no such file in this repo. Could you please tell me how to generate this file? That would be much appreciated!!!
`    def generate_embedding(self, config):
        return pickle.load(open('./tokenizer/embedding.pkl', 'rb')).detach().numpy()`
